pkg/tfvars/ibmcloud/ibmcloud.go

```diff
@@ -6,6 +6,8 @@ import (
 	"github.com/openshift/installer/pkg/tfvars/internal/cache"
 	"github.com/openshift/installer/pkg/types"
 	"github.com/pkg/errors"
+
+	ibmcloudprovider "github.com/openshift/cluster-api-provider-ibmcloud/pkg/apis/ibmcloudprovider/v1beta1"
```

Comment:
nit: The cluster-api-provider import should be in a section before the installer imports.

```suggestion
	"github.com/pkg/errors"
	ibmcloudprovider "github.com/openshift/cluster-api-provider-ibmcloud/pkg/apis/ibmcloudprovider/v1beta1"

	"github.com/openshift/installer/pkg/tfvars/internal/cache"
	"github.com/openshift/installer/pkg/types"
```

Reply:
I moved the CAPI import, but left the `errors` where it was (I assume that we not intended in the suggestion), given the alphanumeric order

Reply:
It would be better for the `errors` import to be in a section above and separate from the installer imports, but that is a very minor nit and well beyond the scope of the BZ.